srml/transaction-payment/rpc/src/lib.rs

```diff
@@ -0,0 +1,105 @@
+// Copyright 2019 Parity Technologies (UK) Ltd.
+// This file is part of Substrate.
+
+// Substrate is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Substrate is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
+
+//! RPC interface for the t
+//! ransa
+//!ction payment module.
+use std::sync::Arc;
+use serde::{Serialize, Deserialize};
+use codec::Codec;
+use client::blockchain::HeaderBackend;
+use jsonrpc_core::{Error, ErrorCode, Result};
+use jsonrpc_derive::rpc;
+use sr_primitives::{
+	generic::BlockId,
+	traits::{Block as BlockT, ProvideRuntimeApi},
+};
+use srml_transaction_payment_rpc_runtime_api::{
+	TransactionPaymentApi as TransactionPaymentRuntimeApi, RuntimeDispatchInfo,
+};
+use self::gen_client::Client as TransactionPaymentClient;
+
+
+/// A wrapper around the request information to query a dispatchable info.
+#[derive(Serialize, Deserialize)]
+pub struct DispatchInfoRequest<AccountId, Extra, Call> {
+	/// Sender aka. origin of the dispatch.
+	pub origin: AccountId,
+	/// The call type.
+	pub call: Call,
+	/// Signed extensions.
+	pub extra: Extra,
+}
+
+#[rpc]
+pub trait TransactionPaymentApi<BlockHash, AccountId, Balance, Call, Extra> {
+	#[rpc(name = "transaction_payment_query_unsigned")]
```

Comment:
The convention is `<namespace>_<camelCasedMethodName>`.

Reply:
Done just to be sure what if namespace is `name_space` or let's stick to `namespace`? for now I named it `payment_queryInfo`